app/assets/stylesheets/components/_mobile-filters.scss

```diff
@@ -109,6 +114,28 @@
     .facets__tags-block {
       display: block;
     }
+
+    // Temporary spacing fix because first item is shorter than the others
```

Comment:
It's unclear what temporary means here - were these changes originally meant to be deployed before the v5 upgrade, and can be removed once the typography changes are deployed?

Reply:
Yes, this has now been changed/removed, it's no longer needed in the previous form.